api/src/test/resources/application.properties

```diff
@@ -6,3 +6,7 @@ spring.jpa.hibernate.ddl-auto=none
 spring.liquibase.enabled=true
 
 api.retry-after.delay=30
+
+api.okta-url=https://dev-418212.okta.com/oauth2/default
```

Comment:
let's set this to `example.com` just to make sure we don't have tests that try to connect to Okta without us realizing that.